### PR TITLE
bt-392: [BUG] Step 4 mobile bugs

### LINKED ISSUE #392: [BUG] Step 4 mobile bugs
**Description:**
Step 4 doesn't correspond to the design

**Environment:**
Android 12

**Steps to reproduce:**
1. Go to Step 4
2. Select Phone number field
3. Click Next button

**Actual Result:**
3. Linked in has unclear hint
4. Phone number validation accepts only Ukrainian +380 numbers

**Expected Result:**
3. LinkedIn has hint as on web "Link format must be https://www.linkedin.com/in/"
4. Phone number validation accepts any country codes and numbers

**Visual Proof (screenshots, videos, text):**
Image
Image

**Severity:**
Major

### PR DESCRIPTION
**Pull request by illia-dobro**
link

plus

num

### CODE REVIEW

File: mobile/src/bundles/talent/enums/contacts-cv-step/cv-and-contacts-form-validation-message.enum.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ const CvAndContactsFormValidationMessage = {
     LINKEDIN_LINK_REQUIRED: 'LinkedIn is required',
     LINKEDIN_LINK_LENGTH:
         'LinkedIn must be between {{#limit.min}} and {{#limit.max}} characters',
-    LINKEDIN_LINK_WRONG_PATTERN: 'Invalid LinkedIn URL format',
+    LINKEDIN_LINK_WRONG_PATTERN:
+        'Link format must be https://www.linkedin.com/in/',
```
Comment (katerynakostikova) [inline mobile/src/bundles/talent/enums/contacts-cv-step/cv-and-contacts-form-validation-message.enum.ts:12]: as I remember PO asked us to skip this part https:// 
Ask QA team about it

**Review by katerynakostikova — CHANGES_REQUESTED** [review]

File: mobile/src/bundles/talent/enums/contacts-cv-step/cv-and-contacts-form-validation-message.enum.ts (hunk 1 above)
Comment (illia-dobro) [reply mobile/src/bundles/talent/enums/contacts-cv-step/cv-and-contacts-form-validation-message.enum.ts:12]: qa

**Comment by mskmee** [comment]
Guess we should have validation schema and enum in the shared layer and use it front/mobile parts, just like we got them to another steps.
Maybe validation schema will be not the same, but enum will be the same anyway.

**Comment by katerynakostikova** [comment]
https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/issues/392#issuecomment-1733123572

**Comment by katerynakostikova** [comment]
See comment

**Comment by katerynakostikova** [comment]
https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/issues/392#issuecomment-1736931442

**Comment by mskmee** [comment]
@illia-dobro

**Comment by katerynakostikova** [comment]
Conflicts

**Review by katerynakostikova — APPROVED** [review]

**Review by mskmee — APPROVED** [review]

**Pull request merged**